packages/tabs/src/TabTitle.tsx

```diff
@@ -125,6 +131,23 @@ const textOverflowStyles = css`
   text-overflow: ellipsis;
 `;
 
+function useDocumentActiveElement() {
+  const [activeEl, setActiveEl] = useState<Element | null>(null);
+
+  const handleFocusIn = useCallback(() => {
+    setActiveEl(document.activeElement);
```

Comment:
Why can't we use `document.activeElement` directly instead of storing it in component state?

If this needs to stay, we should have a comment explaining why it needs to exist here.

Reply:
good catch